### PR TITLE
Githublink

### PR DESCRIPTION
**Pull request by sayanmondal2098**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const app = express()
+
+app.get('/', function (req, res) {
+    res.write('GET request to the homepage');
+    res.write('Hello Server');
+    res.end();    
+  })
+
+
+app.get('/dirname', function (req, res) {
```
Comment (paulodiovani) [inline src/App.js:17]: I'm not following what this is for? :thinking: 

I think we can have only the `/` endpoint for now.

File: src/GetDirectory.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const fs = require('fs');
+const path = require('path');
+const testFolder = path.dirname(require.main.filename)
+
+
+fs.readdirSync(testFolder).forEach(file => {
+  console.log(file);
+});
```
Comment (paulodiovani) [inline src/GetDirectory.js:8]: Do we have any use for this? :confused:

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+var express = require('express')
```
Comment (paulodiovani) [inline src/App.js:1]: Please, add a package.json with `express` and `mongodb` client.

**Review by paulodiovani — CHANGES_REQUESTED** [review]

**Comment by paulodiovani** [comment]
@sayanmondal2098  I'll merge this once the `package.json` is in place and we are able to run with `npm install && npm start` :wink:

File: src/App.js (hunk 1 above)
Comment (sayanmondal2098) [reply src/App.js:17]: the /dirname will send the directory address (pwd) and all files under the folder

File: src/GetDirectory.js (hunk 2 above)
Comment (sayanmondal2098) [reply src/GetDirectory.js:8]: NO :P , just for testing purpose i was created this one

**Comment by sayanmondal2098** [comment]
Issue resolved , i just initialize the module and package,json added . npm start to start the project and npm install to install the dependency . #16

**Comment by paulodiovani** [comment]
Thanks @sayanmondal2098, but I'm felling #17 is a more complete start.

**Pull request closed without merging**